### PR TITLE
Update the CRDs docs

### PR DESCRIPTION
**Pull request by flavio**
Generate the file that holds the description of our CRD.

That was generated by using the `make generate` target inside of the `docs/crds` directory

### CODE REVIEW

**Review by ereslibre — APPROVED** [review]
Oof, forgot about this. LGTM, thanks!

File: docs/crds/README.asciidoc [hunk 1]
```diff
@@ -16,13 +16,15 @@ Package v1alpha2 contains API Schema definitions for the policies v1alpha2 API g
...
+- xref:{anchor_prefix}-github-com-kubewarden-kubewarden-controller-apis-policies-v1alpha2-policyserver[$$PolicyServer$$]
+- xref:{anchor_prefix}-github-com-kubewarden-kubewarden-controller-apis-policies-v1alpha2-policyserverlist[$$PolicyServerList$$]
 
 
 
 [id="{anchor_prefix}-github-com-kubewarden-kubewarden-controller-apis-policies-v1alpha2-clusteradmissionpolicy"]
 ==== ClusterAdmissionPolicy 
 
-ClusterAdmissionPolicy is the Schema for the clusteradmissionpolicies API
+nolint:lll ClusterAdmissionPolicy is the Schema for the clusteradmissionpolicies API
```
Comment (ereslibre) [inline docs/crds/README.asciidoc:27]: The generator did take a nolint comment and wrote it as documentation though... 🤔

**Review by viccuad — COMMENTED** [review]
Should we add a step in the GHA release job, to run that `make generate` and commit the results back?

It seems to be scrapped and updated in https://doc.crds.dev/github.com/kubewarden/kubewarden-controller for example.

File: docs/crds/README.asciidoc (hunk 1 above)
Comment (flavio) [reply docs/crds/README.asciidoc]: Good catch, I fixed that by moving the `//nolint` comment right lower into the block

**Comment by flavio** [comment]
> Should we add a step in the GHA release job, to run that `make generate` and commit the results back?
> 
> It seems to be scrapped and updated in https://doc.crds.dev/github.com/kubewarden/kubewarden-controller for example.

I don't know how hard it would be to do that in a clean way...

**Review by ereslibre — APPROVED** [review]

**Comment by ereslibre** [comment]
Merging, let's move this forward and automate in other PR if desired/feasible

**Pull request merged**